src/pages/Booking/FlyTicket/FinalTicket.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import './FinalTicket.scss';
+
+
+
+class FinalTicket extends Component {
+  constructor() {
+    super();
+    this.state = {
+      ticketCheck: [],
+      ticketdata: ''
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/ticketcheck.json', {
+      method: 'GET'
+    })
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/ticketcheck.json')
```
- public의 내용을 불러오는 경우 로컬호스트와 포트명 생략해줄 수 있습니다
- method 역시 get이 디폴트이기 때문에 생략 가능합니다.